Delivers freebl libraries for sparc.
Should investigate and send upstream.

--- a/mail/installer/package-manifest.in	2017-04-26 15:22:15.253247547 +0000
+++ b/mail/installer/package-manifest.in	2017-04-26 15:22:34.881822175 +0000
@@ -812,7 +812,7 @@
 #endif
 
 ; for Solaris SPARC
-#ifdef SOLARIS
+#ifdef XP_SOLARIS
 @BINPATH@/@DLL_PREFIX@freebl_32fpu_3@DLL_SUFFIX@
 @BINPATH@/@DLL_PREFIX@freebl_32int_3@DLL_SUFFIX@
 @BINPATH@/@DLL_PREFIX@freebl_32int64_3@DLL_SUFFIX@
